modules/create-additional-routes-ocp-oauth.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-user-access/configure-ocp-oauth.adoc
+:_module-type: PROCEDURE
+[id="create-additional-routes-ocp-oauth_{context}"]
+= Creating additional routes for {ocp} OAuth server
+
+When you configure {ocp} OAuth server as an identity provider by using {product-title-short} portal, {product-title} only configures a single route for the OAuth server.
+However, you can create additional routes by using the `spec.customize.annotations` annotation.
+
+.Prerequisites
+
+* You must have configured link:https://docs.openshift.com/container-platform/4.11/authentication/using-service-accounts-as-oauth-client.html[Service accounts as OAuth clients] for your {ocp} OAuth server.
+
+.Procedure
+
+. Create a patch with the customized annotations:
++
+[source,terminal]
+----
+$ CENTRAL_ADDITIONAL_ROUTES='
+spec:
+  central:
+    exposure:
+      loadBalancer:
+        enabled: false
+        port: 443
+      nodePort:
+        enabled: false
+      route:
+        enabled: true
+    persistence:
+      persistentVolumeClaim:
+        claimName: stackrox-db
+  customize:
+    annotations:
+      serviceaccounts.openshift.io/oauth-redirecturi.main: sso/providers/openshift/callback <1>
+      serviceaccounts.openshift.io/oauth-redirectreference.main: "{\"kind\":\"OAuthRedirectReference\",\"apiVersion\":\"v1\",\"reference\":{\"kind\":\"Route\",\"name\":\"central\"}}" <2>
+      serviceaccounts.openshift.io/oauth-redirecturi.second: sso/providers/openshift/callback <3>
+      serviceaccounts.openshift.io/oauth-redirectreference.second: "{\"kind\":\"OAuthRedirectReference\",\"apiVersion\":\"v1\",\"reference\":{\"kind\":\"Route\",\"name\":\"second-central\"}}" <4>
+'
+----
+<1> Redirect URI for setting the main route.
+<2> Redirect URI reference for the main route.
+<3> Redirect URI for setting the second route.
+<4> Redirect URI reference for the second route.
+
+. Apply the `CENTRAL_ADDITIONAL_ROUTES` patch to the Central deployment:
```

Comment:
This patch looks to be targeting the Custom Resource (see `spec.customize`), not deployment. Otherwise, the annotations should be added to the service account.